### PR TITLE
Remove DB from the Client

### PR DESCRIPTION
**Pull request by Krigpl**
It doesn't really belong there.

### CODE REVIEW

File: tests/golem/test_client.py [hunk 1]
```diff
@@ -388,7 +377,6 @@ def test_quit(self, *_):
             app_config=Mock(),
             config_desc=ClientConfigDescriptor(),
             keys_auth=Mock(),
-            database=Mock(),
         )
         self.client.db = None
```
Comment (Wiezzel) [inline tests/golem/test_client.py:381]: This line should be removed as well.

**Review by Wiezzel — APPROVED** [review]
OK, just one more line to delete.

**Review by mfranciszkiewicz — CHANGES_REQUESTED** [review]
The database has been lately moved from `Client` to `Node`, but the application (reactor) shutdown trigger is still set on `Client.quit`, which closes database connections.

**Comment by Krigpl** [comment]
Maybe some other day.

**Pull request closed without merging**